corehq/apps/hqwebapp/static/hqwebapp/js/hq.helpers.js

```diff
@@ -131,6 +131,7 @@ hqDefine("hqwebapp/js/hq.helpers", [
         if (this.length > 1) {
             throw new Error("Multiple elements passed to koApplyBindings");
         }
+        ko.cleanNode(this.get(0)); // make sure we don't apply bindings twice to the same element
```

Comment:
Is there a change in knockout requiring this? The error about applying bindings twice often points to a real bug.